epidose/device/update_filter_d.sh

```diff
@@ -39,21 +39,13 @@ UTIL="$(dirname "$0")/util.sh"
 # Wait until a filter file is required
 wait_till_filter_needed()
 {
-  if [ -r "$FILTER" ] ; then
-    filter_mtime=$(stat -c '%Y' "$FILTER")
-    time_now=$(date +%s)
-    filter_age=$((time_now - filter_mtime))
-    if [ $filter_age -lt $MAX_FILTER_AGE ] ; then
-      log "Filter's age ($filter_age s) is current; no update required"
-      to_sleep=$((MAX_FILTER_AGE - filter_age))
-      log "Sleeping for $to_sleep s"
-      sleep $to_sleep
-      log "Waking up from sleep; new filter is now required"
-    fi
-    log "Filter's age ($filter_age s) makes it stale; update required"
-  else
-    log "No filter available; download required"
-    mkdir -p "$(dirname $FILTER)"
+  # Calculates whether the cuckoo filter is stale or fresh
+  # and return a value accordingly
```

Comment:
So should I create a PR where I migrate the checks found inside the update_filter_d to the util.sh?
We need this because some of the functions found inside update_filter_d are required by the wps_scanner_d.

Reply:
Yes, you can start with one refactoring commit, which doesn't add/change anything

Reply:
Ok, then I will use git rebase to break this comment into two smaller ones.
In the first commit, I will include the migrations of the checks found in the update_filter_d.sh to the util.sh.
In the seconds commit, I will include the WPS-related code.
Ok?

Reply:
Excellent, looking forward!